pkg/cmd/workspace/util/repository_wizard.go

```diff
@@ -29,6 +29,15 @@ func isGitProviderWithUnsupportedPagination(providerId string) bool {
 	}
 }
 
+func isGitProviderAppendsPersonalNamespace(providerId string) bool {
```

Comment:
Rename to `gitProviderAppendsPersonalNamespace`

Reply:
done